src/atomate2/forcefields/utils.py

```diff
@@ -82,18 +130,42 @@ def __init__(self, atoms: Atoms) -> None:
         self.energies: list[float] = []
         self.forces: list[np.ndarray] = []
         self.stresses: list[np.ndarray] = []
+
+        self._store_magmoms = True
+        self.magmoms: list[np.ndarray] = []
+
         self.atom_positions: list[np.ndarray] = []
         self.cells: list[np.ndarray] = []
 
+        self._store_md_outputs = store_md_outputs
+        if self._store_md_outputs:
+            self.velocities: list[np.ndarray] = []
+            self.temperatures: list[float] = []
+
```

Comment:
i think i agree with this design but maybe worth discussing whether `self.velocities` and `self.temperatures` should always be initialized and just left empty if `store_md_outputs=False`?

Reply:
Changed as well, these are always initialized but no data is added to them, and no info is saved to the trajectory if an MD run is not performed